capa/features/extractors/vmray/extractor.py

```diff
@@ -87,11 +92,11 @@ def extract_call_features(
         self, ph: ProcessHandle, th: ThreadHandle, ch: CallHandle
     ) -> Iterator[Tuple[Feature, Address]]:
         yield from capa.features.extractors.vmray.call.extract_features(ph, th, ch)
-
-    def get_call_name(self, ph, th, ch) -> str:
-        # TODO (meh): https://github.com/mandiant/capa/issues/2148
-        raise NotImplementedError()
-
+    
+    def get_call_name(ph, th, ch) -> str:
```

Comment:
See capa's [CAPE extractor implementation](https://github.com/mandiant/capa/blob/b617179525b86297e2541722e7a8832a49df59e2/capa/features/extractors/cape/extractor.py#L87-L122) as a guide. This method should return a human-readable representation of a call that includes the name of the call, arguments, return value, etc.

Reply:
Leaving get_call_name as is per conversation during Thursday's meeting.

Reply:
can you please detail the discussion/why this should differ from the cape implementation (here and in the code)?